src/js/pensions/config/form.js

```diff
@@ -783,13 +792,180 @@ const formConfig = {
                   properties: {
                     relationship: dependents.items.properties.relationship,
                     fullName: dependents.items.properties.fullName,
-                    childDateOfBirth: dependents.items.properties.childDateOfBirth
+                    childDateOfBirth: dependents.items.properties.childDateOfBirth,
+                  }
+                }
+              }
+            }
+          }
+        },
+        childrenInformation: {
+          path: 'household/dependents/child/information/:index',
+          title: item => `${item.fullName.first} ${item.fullName.last} information`,
+          showPagePerItem: true,
+          arrayPath: 'dependents',
+          itemFilter: (item) => !!(item.relationship === 'child'),
+          schema: {
+            type: 'object',
+            properties: {
+              dependents: {
+                type: 'array',
+                items: {
+                  type: 'object',
+                  properties: {
+                    childPlaceOfBirth: dependents.items.properties.childPlaceOfBirth,
+                    childSocialSecurityNumber: dependents.items.properties.childSocialSecurityNumber,
+                    childRelationship: {
+                      type: 'string',
+                      'enum': [
+                        'biological',
+                        'adopted',
+                        'stepchild'
+                      ]
+                    },
+                    attendingCollege: dependents.items.properties.attendingCollege,
+                    disabled: dependents.items.properties.disabled,
+                    previouslyMarried: dependents.items.properties.previouslyMarried,
+                    married: dependents.items.properties.married,
                   }
                 }
               }
             }
+          },
+          uiSchema: {
+            dependents: {
+              items: {
+                'ui:title': createDisclosureTitle('fullName', 'Information'),
```

Comment:
Thanks @jbalboni, have renamed to `createHouseholdMemberTitle`, please let me know if you recommend a different option.